src/styles/main.scss

```diff
@@ -1,3 +1,12 @@
+* {
+  box-sizing: border-box;
+}
+
 body {
   margin: 0;
+  padding: 20px;
 }
```

Comment:
I think it is better to do with compilation "reset file", but the code works and looks nice with BEM condition.
Nice job!=)